src/LiveComponent/assets/src/Component/plugins/LoadingPlugin.ts

```diff
@@ -15,33 +16,33 @@ interface ElementLoadingDirectives {
 
 export default class implements PluginInterface {
     attachToComponent(component: Component): void {
-        component.on('loading.state:started', (element: HTMLElement, request: BackendRequest) => {
-            this.startLoading(element, request);
+        component.on('loading.state:started', (element: HTMLElement, request: BackendRequest, component: Component) => {
+            this.startLoading(component, element, request);
```

Comment:
We already have the `component` as the argument to `attachToComponent` and we're attaching the `loading.state.started` to that specific component. So o need for the new argument to the callback: we can use the `component` variable already

Reply:
As i could not test, i did it in a defensive way :) 

I'll remove it

Reply:
Fix / rebased / rebuilded (style cannot test locally -- will fix this next week)

Reply:
Think i did what you meant but not sure